README.md

```diff
@@ -85,8 +95,9 @@ Rails.application.config.to_prepare do
 end
 ```
 
-The initializer registers two `Valkyrie::MetadataAdapter` instances for storing metadata:
+The initializer registers three `Valkyrie::MetadataAdapter` instances for storing metadata:
```

Comment:
@tpendragon When both of your PRs get merged this will be Four not two.  I am guessing since you are changing the same line git will yell...